tools/arcilator/arcilator.cpp

```diff
@@ -68,89 +70,97 @@ using namespace arc;
 // Command Line Arguments
 //===----------------------------------------------------------------------===//
 
-static cl::OptionCategory mainCategory("arcilator Options");
+static llvm::cl::OptionCategory mainCategory("arcilator Options");
```

Comment:
Changing namespaces and using is not that far off from a whitespace change.  If you are going to do it, please do it in a separate PR to avoid noise.

Reply:
The reason I am doing it here is because the execution engine import introduces a name conflict if both the `mlir` and `llvm` namespaces are used. Hence why I had to drop one, and I picked llvm because it seemed to be the one most unrelated. Is a second PR really necessary? It is very much related to the meaningful content of this PR.